url_filter/integrations/drf.py

```diff
@@ -87,6 +87,23 @@ def get_filter_class(self, view, queryset=None):
         if filter_class:
             return filter_class
 
+        if filter_fields == '__all__':
```

Comment:
thanks for the contribution!

the intent here is great however it seems to be duplicating the if statement below very closely. easier change might be to add support in `ModelFilterSet` to support `__all__` as a valid value. from the quick look only required change will be in

https://github.com/miki725/django-url-filter/blob/ba476094adeb0db4bb39ee0f4c3d49681d1bd1ba/url_filter/filtersets/base.py#L470

to support both `None` and `__all__`:

```suggestion
if self.Meta.fields in [None, '__all__']:
```

let me know if you will need any help with tests or anything else.

thanks again

Reply:
your quite welcome!  

it is duplicating the if statement pretty closely, I tried to pass None in for my ModelViewSet's filter_fields and I was not able to filter on any of the columns.  Thus why I changed what I changed.  The below didn't work (please let me know if I'm using this correctly)
```
class ParamGroupScpiCommandsViewSet(ModelViewSet):
    serializer_class = ParamGroupScpiCommandsSerializer
    queryset = ParamGroup_ScpiCommands.objects.all()
    filter_backends = [DjangoFilterBackend]
    filter_fields = None

```
but after the code edit that I submitted 

```
class ParamGroupScpiCommandsViewSet(ModelViewSet):
    serializer_class = ParamGroupScpiCommandsSerializer
    queryset = ParamGroup_ScpiCommands.objects.all()
    filter_backends = [DjangoFilterBackend]
    filter_fields = '__all__'
```

works

I left the original code in tact as to not corrupt anyone's existing filters that they have set up.  

Side note, I'm totally going to need help writing tests lol

Reply:
I just edited this area in the code, the interesting part here is when the filterset gets created using None the primarykey field (Django id column) doesn't get added to the filterable fields.  This is why I thought my solution worked correctly.  After changing using your logic, the id field is still non existent.  I love this library and have just been trying to get around to helping out where I can so thank you for your support and great library!

Reply:
I checked in another change to the same branch in my repository but I don't know why its not showing that new commit

Reply:
> I love this library and have just been trying to get around to helping out where I can so thank you for your support and great library!

awesome. glad you enjoy it 😄 

and thank *you* for helping out. any contribution is welcome.